plugins/postgres/dbt/include/postgres/macros/utils/columns_spec_ddl.sql

```diff
@@ -1,10 +0,0 @@
-{% macro get_column_names() %}
```

Comment:
Between the v1.5.0 and next v1.5.x patch release: A Postgres user could have called this macro by name (`get_column_names`), and it will no longer work because now we're defining a default and dispatching it. I'm not worried about the likelihood of that happening (this is an internal & undocumented macro), just mentioning as a thing that theoretically could happen